src/ZombieDriver.php

```diff
@@ -94,13 +94,14 @@ public function stop()
     public function reset()
     {
         $js = <<<JS
-browser.destroy();
-browser = null;
```

Comment:
Please describe the actual test which is failing for you because of the popup being the active window in your Selenium tests (the best way is writing it as a series of Mink calls, as this can become a test in the driver testsuite easily). And make sure that this test relies on supported usages of Mink methods (otherwise I would not consider it a bug).